tools/check_test.js

```diff
@@ -30,6 +30,12 @@ function Driver() {
     fail: 0,
     skip: 0,
     timeout: 0,
+    json: {
+      bin: { text:0, total:0, data:0, bss:0, rodata:0 },
```

Comment:
@kolipka I think that the most critical point in the current way is to run the tests using `eval`. Some incorrect states after executing a test could affect the followling tests. I guessed that were already informed in https://github.com/Samsung/iotjs/issues/1090.